src/main/java/ui/Ui.java

```diff
@@ -1,52 +1,104 @@
 package ui;
 
+import language.Language;
+import language.LanguageFactory;
+import language.Languages;
 import move.Moves;
 
 import java.io.InputStream;
 import java.io.PrintStream;
+import java.util.Arrays;
+import java.util.List;
 import java.util.Scanner;
 
 public class Ui {
 
+    final static private String NEWLINE = "";
+    private Language language;
     private PrintStream output;
     private Scanner input;
 
-    public Ui(PrintStream output, InputStream input) {
+    public Ui(PrintStream output, InputStream input, Language language) {
         this.output = output;
         this.input = new Scanner(input);
+        this.language = language;
+    }
+
+    public void askForLanguage() {
+        output.println("Please choose a language:");
```

Comment:
It's a bit inconsistent to hard code the English strings in here. This should probably also be a `language.chooseLanguage()` or something like that.

Reply:
Would it make sense for the initial prompt to say `Please choose a language/Scegli una lingua`? Incase the Italian person doesn't understand English?

Reply:
@RabeaGleissner yep, as I said in my comments when I slacked you the link to PR, this is something I'm working on! 
@gemcfadyen Yup, working on that

Reply:
Hey @gemcfadyen , how did you know the Italian for "choose a language"?? 😄